Sources/SwiftUITooltip/TooltipViewExtension.swift

```diff
@@ -8,27 +8,27 @@
 import SwiftUI
 
 public extension View {
-    func tooltip<TooltipContent: View>(@ViewBuilder content: @escaping () -> TooltipContent) -> some View {
+    func tooltip<TooltipContent: View>(isEnabled: Binding<Bool>, @ViewBuilder content: @escaping () -> TooltipContent) -> some View {
```

Comment:
I think all of these should come in addition to the existing `.tooltip`'s. 

Another thing is I think the `isEnabled` could be shortened to `enabled` without making it any harder to understand from developer side 🤔

Also I was thinking of using the `_ isEnabled: Binding<Bool>` argument to allow just passing the variable wihout the argument name. This would of course have ti be balanced with the side parameter, since it can also not have the argument name.

Reply:
Acknowledged. Renamed `isEnabled` to `enabled`. I believe `enable` doesn't make for a good argument name because it's a verb and is better suited as a function name. Also moved `enabled` as the initial argument in the extension to remove the label requirement.

Reply:
My understanding of your first request is to be backwards compatible with previous versions so I added a default value of `.constant(true)` to the `enabled` argument.

Reply:
Yes, my first request was to keep the backward compatibility, but `.tooltip` functions in the `View` modifier are still overriding the old functions. If someone had a function that just said `.tooltip { Text("Something") }` then it will be broken now and they will have to manually add the `enabled` argument for their code to work again.

Reply:
Also, I think setting the `enabled` to a constant value in the `TooltipModifier` doesn't make sense since you have the boolean binding, right?

Reply:
With the current implementation, `.tooltip { Text("Something") }` will continue to work as expected. I added additional function definitions so that the follow will also work:
```
Text("How much?")
                    .tooltip(.leading) {
                        Text("Just the tip")
                    }
```

Reply:
Nice, then this one is resolved!